pkg/cmd/server/api/types.go

```diff
@@ -968,6 +968,11 @@ type KubernetesMasterConfig struct {
 	// the server will not start. These values may override other settings in KubernetesMasterConfig which may cause invalid
 	// configurations.
 	ControllerArguments ExtendedArguments
+
+	// ReconcileMasterEndpoints indicates that OpenShift should update the list of endpoints for the
+	// "kubernetes" Service, removing any entry from the list whose corresponding master/apiserver is
+	// unhealthy or down.
+	ReconcileMasterEndpoints bool
```

Comment:
For origin it's fine with enabled by default.  Bake time I was referring to was giving Dan warm fuzzies given the kube 1.3 release train, rather than any doubt that we can make this work.